spec/factories/events.rb

```diff
@@ -4,6 +4,10 @@
         sequence(:description) { |n| "description#{n}" }
         sequence(:discipline) { |n| "discipline#{n}" }
         player_type Event.player_types[Event.player_types.keys.sample]
+        game_mode League.game_modes[League.game_modes.keys.sample]
```

Comment:
Earlier, I wrote that Event should be somehow an abstract class, i.e. not valid when generated alone. Aren't we destroying this concept, when using the League game-modes to generate a valid Event class?

Reply:
again this is also mostly so that constantins tests still run it's probably a good idea to move those tests to the league model though, I agree.